woocommerce/wc-admin
# Change product_gross_revenue to product_net_revenue

Fixes #1072 

Updates the products table and column/property naming to reflect actual values that are net and not gross.

### Screenshots
<img width="1472" alt="screen shot 2018-12-17 at 6 11 02 pm" src="https://user-images.githubusercontent.com/10561050/50080679-9b0fe100-0227-11e9-9d9b-cd8b37eb9cad.png">

### Detailed test instructions:

1. You may need to delete your `wc_order_product_lookup` table or manually modify the `product_gross_revenue` column to be `product_net_revenue` to test this.
2. Open the Product report page.
3.  Check that "Gross Revenue" has been replaced by "Net Revenue."
4.  Check that the Revenue report page has not been affected since it contains both gross and net revenue reporting.
